doc/dev/testing.xml

```diff
@@ -51,29 +51,20 @@ gap> if LoadPackage( "ctbllib" ) &lt;&gt; fail then
 Moreover, it is recommended to group tests that require packages to be 
 performed after all other tests are completed.
 
-Some of the test files in the <F>tst</F> directory are listed in the file
-<F>tst/testinstall.g</F>,
-running this subset of tests is recommended after installing &GAP;,
+The directory <F>tst/testinstall</F> contains a subset of tests which are 
+recommended after installing &GAP;,
 see <URL>http://www.gap-system.org/Download/INSTALL</URL>.
 The idea is that the examples in these <C>.tst</C> files are typical
 &GAP; computations,
 and that running these tests requires only a few minutes.
-Each of the <C>.tst</C> files in this subset must contain the string
-<C>"To be listed in testinstall.g"</C> in its header,
-in order to make it possible to create <F>tst/testinstall.g</F> automatically
-from an existing version of this file and the contents of the <F>tst</F>
-directory, see section <Ref Sect="Sect-TestSuite"/>.
+The &GAP; script <F>tst/testinstall.g</F> will run the tests in 
+<F>tst/testinstall</F>.
 <P/>
 
-Additionally, all test files in the <F>tst</F> directory are listed in the file
-<F>tst/testall.g</F> which can be used to run all tests instead of only a subset
-of those (see section <Ref Sect="Sect-TestSuite"/>).
+Further tests are contained in <F>tst/teststandard</F>. The script 
+<F>tst/teststandard.g</F> will run all tests.
```

Comment:
"all" - i.e both from `tst/testinstall` and `tst/teststandard`? I suggest to clarify that it will look into the other dir too.